scripts/thompson_sampling_bernoulli.py

```diff
@@ -0,0 +1,66 @@
+# Resolution of a Multi-Armed Bandit problem
+# using Thompson Sampling.
+# Author: Gerardo Durán-Martín (@gerdm)
+
+import jax
+import jax.numpy as jnp
+import matplotlib.pyplot as plt
+import pyprobml_utils as pml
+from jax import random
+from jax.nn import one_hot
+from jax.scipy.stats import beta
+
+def thompson_sampling_step(state, key):
```

Comment:
I suggest a slightly more general API (as a warmup to the contextual case :).
Something like this.
```
def thompson_sampling_step(key, bel, true_reward, predict_reward):
  params = bel.sample(key)
  action = predict_reward(params).argmax()
  reward = true_reward(action)
  bel = bel.update(action, reward)
  return bel
```

You can then implement bernoulli bandit and Gaussian bandits with the same API, but diffferent update functions. Contextual bandits will be a small extension (future PR).